recipes/copulae/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "copulae" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4b6963c488ebec36fb91fb5cabd07c14a4d1c596df11c4865dd3b49a549157b8
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script:  "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - numpy 1.14.*  # [py==37]
+    - numpy 1.12.*  # [py!=37]
+    - pandas >=0.23
+    - pip
+    - python
+    - pytest-cov
+    - scipy >=1.1
+    - statsmodels >=0.9
+  run:
+    - {{ pin_compatible('numpy') }}
```

This is only needed when the package contains compiled code. Looking at the `setup.py` it looks like you want:
```suggestion
    - numpy >=1.15
```

---

Okay! Changing this. Thanks for the info!